src/main/java/teammates/ui/controller/InstructorCourseRemindAction.java

```diff
@@ -66,7 +66,14 @@ public ActionResult execute() throws EntityDoesNotExistException {
 
             statusToUser.add(new StatusMessage(Const.StatusMessages.COURSE_REMINDER_SENT_TO + studentEmail,
                                                StatusMessageColor.SUCCESS));
-            redirectUrl = Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE;
+
+            /*see and compare if the request is coming from instructorCourseDetails
+            or instructorStudentList and redirect accordingly */
+            if (compareRefererAndCurrentPage(Const.ActionURIs.INSTRUCTOR_COURSE_DETAILS_PAGE)) {
```

Comment:
the method name is confusing. It doesn't tell me it's supposed to return a boolean, even though it does return a boolean.